docs/data/ingest-library/developer_guide/architecture.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: Architecture
+sidebar_position: 0
+---
+
+![Insert architecture diagram]
+
+## Package Structure
+
+The ingest library is broadly split into three main packages:
+
+1. **Ledger Backends**: A wrapper around sources of ledger data to fetch ledgers from the Stellar network.
```

Comment:
I'm not sure if wrapper is the right term to use. I would suggest rephrasing the description to something like
```suggestion
1. **Ledger Backends**: Used to stream ledgers from the Stellar network.
```